docs/tasks/2_extend_arbitration_graph.md

```diff
@@ -72,6 +72,7 @@ explicit PacmanAgent(const entt::Game& game)
     // Add the ChaseGhost behavior component to the priority arbitrator (before the AvoidGhost behavior component!)
     rootArbitrator_->addOption(chaseGhostBehavior_, PriorityArbitrator::Option::Flags::INTERRUPTABLE);
     rootArbitrator_->addOption(avoidGhostBehavior_, PriorityArbitrator::Option::Flags::INTERRUPTABLE);
+    rootArbitrator_->addOption(eatClosestDotBehavior_, PriorityArbitrator::Option::Flags::INTERRUPTABLE);
```

Comment:
This was missing because it was previously added in an additional tutorial task that has since been removed.